test/integration_test.go

```diff
@@ -44,10 +46,21 @@ var id string
 var externalIdentifier string
 var tacoIdentifier string
 
+func clearDatabase() {
```

Comment:
What is the reason to clear the database during this test?

Reply:
If you don't clear the database, then we can't insert a record that has the same source id of any other that we've previously used.

Reply:
I see that, but note that `example/request.json` used in the test doesn't have `sourceId` moved up to the top level so it's not actually hit here. My preference here would be to use existing code (i.e. `DeleteAllversions`) vs. something that clears out the db. While unlikely, it's possible that one could run the integration test against taco-demo and clear out that db. I'll defer to the group input here so we can get it in however.

Reply:
I've updated `example/request.json` to have a `dedupeIdentifier` now, so I think the code *is* required.  Having a `dedupeIdentifier` that matches `identification.sourceId` should match the real use case.